stable/database/templates/deployment.yaml

```diff
@@ -24,8 +24,16 @@ spec:
     type: RollingUpdate
   template:
     metadata:
-      {{- if .Values.database.podAnnotations }}
+      {{- with .Values.database.te.nodeSelector }}
```

Comment:
Why was this moved up here?

Reply:
This was a mistake.  I've put it back to where it belongs.